CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Print values that are not object specifiers with `clojure.core/prn`.
+- Print all values, including object specifiers, as EDN.
```

Comment:
A more correct way of saying this is "print readably", it's not always EDN, e.g.:

```
(prn #"foo")
#"foo"
```
A regex isn't EDN, but this output is readable by the Clojure reader.

Reply:
Ah, great. Will fix.